stages/org.osbuild.grub2.iso.meta.json

```diff
@@ -44,6 +44,11 @@
             }
           }
         },
+        "efi_src_dir": {
+          "type": "string",
+          "description": "The source path to use for the EFI/ binaries when installing is set to True",
```

Comment:
What does "when installing is set to True" refer to?

Reply:
Silly me, I fixed the wrong test, I will also fix the same issue in the org.osbuild.grub2.meta.json file